.codeclimate.yml

```diff
@@ -11,6 +11,7 @@ checks:
       threshold: 250 # not number of lines, but mass threshold (see default analysis configuration)
 
 exclude_patterns:
-- "**/test/"
-- "**/androidTest/"
-- "**test"
+  - "**/test/"
+  - "**/androidTest/"
+  - "**test"
+  - "**/di/" # Exclude Hilt modules (Dependency Injections)
```

Comment:
We shouldn't exclude anything according to the email we just received :
_"The line coverage of your tests must be at least 80% of the total code (without any code exclusion)"_
```suggestion
```

Reply:
Ok, I did this because they are already excluded from jacoco